src/pages/create-or-edit-question/CreateOrEditQuestion.tsx

```diff
@@ -106,7 +106,10 @@ const CreateOrEditQuestion: React.FC = () => {
   } = useForm<CreateOrEditQuestionForm>({
     initialValues: initialValues,
     onSubmit: () => {
-      id ? editQuestion({ ...data, id: id }) : createQuestion(data)
+      if (id) {
+        editQuestion({ ...data, id: id })
+      }
+      createQuestion(data)
```

Comment:
```suggestion
      if (id) {
        editQuestion({ ...data, id: id })
      } else {
        createQuestion(data)
      }
```
In your current implementation, if `id` is a truthy value, it will edit the question and then create a new one. I believe that is not what you want to do.

Reply:
Also, consider extracting this logic to a separate `handleSubmit` function.